packages-web/signature/package-lock.json

```diff
@@ -17,9 +17,9 @@
             "dev": true
         },
         "@types/react": {
-            "version": "16.9.2",
-            "resolved": "https://registry.npmjs.org/@types/react/-/react-16.9.2.tgz",
-            "integrity": "sha512-jYP2LWwlh+FTqGd9v7ynUKZzjj98T8x7Yclz479QdRhHfuW9yQ+0jjnD31eXSXutmBpppj5PYNLYLRfnZJvcfg==",
+            "version": "16.8.25",
```

Comment:
Looks like react type version has been downgraded, could you check if its really the case by removing node_modules folder completely and see this happens again ?

Reply:
I deliberatly downgraded the react type dependency version, because it was conflicting with the react type dependency version defined in the root of the repo.

Reply:
It gave duplicate errors when building the widget and I think the root of mono repo is leading. I think we align this root dependency with what is provided by the Mendix client (React is an external dependency in the bundling process).